docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nginx:
-    image: nginx:alpine
+    image: fabiocicerchia/nginx-lua:1.27.2-almalinux9.4-20240923
```

Comment:
Nit: can we use our own Dockerfile simliar to [this one](https://stackoverflow.com/a/55994069)?